views/view_package.dt

```diff
@@ -64,20 +64,13 @@ block body
 
 		p= versionInfo["description"].opt!string
 		br
-		p To use this package, put the following dependency into your project's dependencies section:
+		p To use this package, run the following command in your project's root directory:
 		.clipboard-pkg
 			- string expr = versionInfo["version"].get!string;
 			- if( !expr.startsWith("~") ) expr = "~>" ~ expr;
 			.clipboard-pkg-row
-				.clipboard-pkg-title dub.json
-				input(id="package-clipboard-json",value='"#{packageName}": "#{expr}"')
-				button(class="btn-clipboard",data-clipboard-target="#package-clipboard-json")
-					img(src="/images/clippy.svg", width=13, alt="Copy to clipboard")
-
-			.clipboard-pkg-row
-				.clipboard-pkg-title dub.sdl
-				input(id="package-clipboard-sdl",value='dependency "#{packageName}" version="#{expr}"')
-				button(class="btn-clipboard",data-clipboard-target="#package-clipboard-sdl")
+				input(id="package-clipboard",value='dub add #{packageName}')
```

Comment:
Shouldn't we include the version here?

Reply:
`dub add` will query the registry for the latest version. `dub add packageName=1.2.3` would do the same thing as just `dub add packageName`

Reply:
Yes, but by supplying the exact version we can make sure the user gets an error if there's an issue with the registry mirror he's using. Also currently we advertise the current version:

![image](https://user-images.githubusercontent.com/4370550/53048614-71786d80-3495-11e9-9ddc-2c6e7a95f509.png)

I had a quick look around. Cargo does it the same way, but PIP and NPM also don't use the exact version. Other opinions?